examples/datachannel-filexfer/README.rst

```diff
@@ -1,27 +1,37 @@
 Data channel file transfer
 ==========================
 
+- ``Important``:
+
+  - Copy and paste signaling example can not work on Windows python environment due to restricton of asyncio.
+
+  - To try file transfer example on windows environment, please see `websocket signaling version`_.
+
 This example illustrates sending a file over a data channel using an
 RTCPeerConnection and a "copy and paste" signaling channel to exchange SDP.
 
+..  _websocket signaling version: https://github.com/aiortc/aiortc/blob/master/examples/datachannel-filexfer/README_WS_SIGNALING_VERSION.rst
+
+
 First install the required packages:
 
 .. code-block:: console
 
     $ pip install aiortc uvloop
 
+
 To run the example, you will need instances of the `filexfer` example:
 
 - The first takes on the role of the offerer. It generates an offer which you
   must copy-and-paste to the answerer.
 
 .. code-block:: console
 
-   $ python cli.py send somefile.pdf
+   $ python filexfer.py send somefile.pdf
```

Comment:
This is a good point, I've merged it into master

Reply:
>You can also remove the text about needing to use websockets on Windows, there was already support for udp + TCP.
there is no description about udp+TCP signaling.
How can i describe about websocket signaling?

like below?
- copy and paste signaling can not work on windows platform
- there is websocket signaling method and it can work at windows platform
- there are other sgnaling methods which can work on windows (but i do not know how to use the mthods to filexfer example...)

Reply:
Again, you can control which signaling method is used with command line arguments, so the fact that copy-and-paste doesn't work is irrelevant.

You are right that the additional signaling methods need documentation but here is a start. If sender / receiver are on the same machine all you need is:

**Sender**  (will listen on 127.0.0.1 port 1234 for signaling)

`python filexfer.py -s tcp-socket send somefile.pdf`

**Receiver** (will connect to 127.0.0.1 port 1234 for signaling)

`python filexfer.py -s tcp-socket receive received.pdf`

If the sender / receiver are on different machines, add `--signaling-host <THE IP OF THE SENDER>`.

Reply:
@jlaine 
I fixed rst files.
but detail of tcp-socket signaling is not described.
I only added the link to above your post.
( I am not good at rst style markup. so I couldn't add above content to README.rst